src/test/java/de/mediathekview/mserver/crawler/srf/tasks/SrfSendungenOverviewPageTaskTest.java

```diff
@@ -44,7 +46,7 @@ public SrfSendungenOverviewPageTaskTest(String aHtmlFile, int aExpectedUrls) {
   @Test
   public void test() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)